lib/model.js

```diff
@@ -959,6 +959,10 @@ Model.prototype.sync = function(options) {
     , attributes = this.tableAttributes;
 
   return Promise.try(function () {
+    if (options.hooks) {
```

Comment:
In the top of `sync` options is copied from the model options, you actually have to make sure that hooks is then removed and it uses the boolean value of the directly passed options instead.